tools/starman.psgi

```diff
@@ -6,7 +6,8 @@ BEGIN {
 
 package LedgerSMB::FCGI;
 
-use lib 'lib';
+use FindBin;
+use lib $FindBin::Bin . "/../lib";
```

Comment:
Yes I did test it.   It runs relative to the starman.psgi file.
One serious issue is that the directory that you run this now must have a valid ledgersmb.conf file in it but that is a separate problem really.